x-pack/test/fleet_api_integration/apis/epm/setup.ts

```diff
@@ -24,8 +24,10 @@ export default function (providerContext: FtrProviderContext) {
     describe('setup performs upgrades', async () => {
       const oldEndpointVersion = '0.13.0';
       beforeEach(async () => {
+        const url = '/api/fleet/epm/packages/endpoint';
+        await supertest.delete(url).set('kbn-xsrf', 'xxxx').send({ force: true }).expect(200);
```

Comment:
this was failing while trying to force install `0.13.0` onto latest (`1.3.0-dev1`) so explicitly delete installed version so can install clean.